packages/@o3r/testing/project.json

```diff
@@ -41,11 +46,33 @@
     "build-tools": {
       "executor": "nx:run-script",
       "outputs": ["{projectRoot}/dist/src/tools/protractor"],
+      "inputs": [
+        "base-build",
```

Comment:
should `tsconfig.build.tools.json` be added here?
(same for `tsconfig.build.utils.json` bellow)